extension/webapp/src/main/webapp/WEB-INF/conf/platform/application-registry-configuration.xml

```diff
@@ -161,38 +161,6 @@
               </field>
             </object>
           </object-param><object-param>
-            <name>development</name>
```

Comment:
Is there any impact for existing customers ?
We think that this gadget is not used so it is not a problem to remove it, but is there any impact on the application registry or on the portlets catalog (when editing a page) for customers who have this gadget declared ? Did you do the test with a 4.3 migrated dataset ?